cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -18,7 +18,7 @@ allowHostNetwork: false
 # set to true if running rook with the provider as host
 allowHostPorts: false
 priority:
-allowedCapabilities: []
+allowedCapabilities: ["MKNOD"]
```

Comment:
With this change, can we revert #9191 that still sets the osd init containers to privileged?

Reply:
Actually, they are different but can be confusing. I believe OCP still needs a privileged container for some reason. At the same time, we must declare which capabilities pods are allowed to acquire at runtime, this is what this patch is made for. I'll hopefully revisit at some point why we need a privileged container to do the block copy. For now, we just need both I believe.